package.json

```diff
@@ -86,9 +90,17 @@
     "webpack-hot-middleware": "^2.12.2"
   },
   "jest": {
-    "testRegex": "(/test/.*\\.spec.js)$",
+    "testRegex": "(/test/unit/.*\\.spec.js)$",
     "setupFiles": [
-      "test/env.js"
+      "test/unit/env.js"
+    ],
+    "collectCoverageFrom": [
```

Comment:
Is there any way to blackist the tests directory and leave all the others? Whitelisting files will always lead to us forgetting to whitelist some new directory/file when it becomes available. I want to reverse that.